lld/ELF/Driver.cpp

```diff
@@ -1507,6 +1507,9 @@ static void readConfigs(Ctx &ctx, opt::InputArgList &args) {
   setUnresolvedSymbolPolicy(ctx, args);
   ctx.arg.power10Stubs = args.getLastArgValue(OPT_power10_stubs_eq) != "no";
 
+  if (ctx.arg.osabi == ELFOSABI_OPENBSD)
```

Comment:
osabi is not set here. It is first set by the OPT_m code below, then possibly inferred by input files or read from linker script OUTPUT_FORMAT.

It seems that we have to read it unconditionally here